scripts/zones/Korroloka_Tunnel/IDs.lua

```diff
@@ -26,6 +26,10 @@ zones[xi.zone.KORROLOKA_TUNNEL] =
         LEFT_SPRING_EARLY             = 7340,  -- You are not warm enough yet. You will need to spend more time than that in the spring to get your body heated up.
         LEFT_SPRING_CLEAN             = 7341,  -- Your whole body is piping hot, and the smell of the Rafflesia pollen is gone!
         MORION_WORM_1                 = 7344,  -- It appears to be a hole made by some kind of animal. Fragments of iron ore are scattered around the area...
+        FILL_FLASK                    = 7343,  -- You carefully draw the water stored in the clam into <keyitem>
+        STILL_LIGHT                   = 7344,  -- The Flask still feels light...
```

Comment:
It appears that there might be an issue with the identifiers MORION_WORM_1 and STILL_LIGHT, as they both have the value 7344.

Reply:
This is likely due to ID shifts and they need to be updated to current retail ids

Reply:
Good call. I didn't learn this was a thing until a few hours ago. Will be fixed momentarily.